Update managing logs page in Lisk docs (#1904)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/run-blockchain/logging.adoc

:url_bunyan: https://github.com/trentm/node-bunyan
:url_bunyan_docs: http://trentm.com/node-bunyan/bunyan.1.html

> For monitoring or debugging your node, the Lisk SDK v6 tracks all activities that occur within the node by creating applicable log messages.

Reviewer: Please check, if the numbers (10-60) are still used to identify log levels, or if they can be removed.

Author: Yes, they are still valid and correct 👍

|Trace(10) |Logging from external libraries used by your node, or otherwise very detailed application logging.
|===

> By using the following config setting listed in the snippet below, it is possible to set the log level of the app.

Reviewer: ```suggestion
To change the node log level, update the `logLevel` property in the `config.json` file, under xref:v6@lisk-sdk::config.adoc#system[system.logLevel] as listed in the snippet below.
```

> 
> [source,bash]

Reviewer: ```suggestion

.`system` options in config.json
[source,json]
```

----
system: {
dataPath: '~/.lisk/beta-sdk-app',
version: '0.1.0',
keepEventsForHeights: 300,
logLevel: 'info',
enableMetrics: false,
backup: {
height: 0,
},
}
}

> ----

Reviewer: Please add indentation.

<1> Pretty-prints console logs with log level equal or higher to the console log level.

== Logging to a file

// [[file_log_stream]]
// === File log stream
// Managing logs ::

All log file support has been removed from the Lisk SDK as the application `stdout output` can be simply output to a file by a user as required.
For example, on Ubuntu and MacOS you can already use the CLI options to output to a file as shown below.
----

> ./bin/run start  > app.log will output stdout to a file app.log

Reviewer: the part after app.log seems to be a comment
```suggestion
./bin/run start  > app.log # will output stdout to a file app.log
```

----
To continue on appending in the same file run the following command.
----

> ./bin/run start  >> app.log

Reviewer: Looks like the same command as above, can be removed.
